lib/rake/circle.rake

```diff
@@ -195,21 +195,15 @@ def test_eyes?
 end
 
 def start_sauce_connect
-  # Use latest sauce connect client for each run so we don't have to keep up with updates and end-of-lifes.
-  # If a newly-released version breaks the build, a quick fix to unblock the issue is to temporarily
-  # pin the version we use to the last working version, while we schedule the task to get the upgraded version
-  # working. You can do this by replacing `sc_download_url` with a hard-coded download url.
-
-  sc_version_info = JSON.parse(Net::HTTP.get(URI('https://saucelabs.com/versions.json')))
-  sc_download_url = sc_version_info['Sauce Connect']['linux']['download_url']
+  sc_download_url = 'https://saucelabs.com/downloads/sauce-connect/5.2.1/sauce-connect-5.2.1_linux.x86_64.tar.gz'
```

Comment:
Unfortunately the Sauce Connect v5 team broke many things (pointlessly?), one of which is they stopped updating `https://saucelabs.com/versions.json`, and provide no equivalent, so now we have to hardcode the URL to download and it isn't evergreen.